apps/admin-gui/src/app/shared/components/two-entity-attribute-page/two-entity-attribute-page.component.ts

```diff
@@ -62,21 +61,32 @@ export class TwoEntityAttributePageComponent implements OnInit {
     this.setMessages(this.secondEntity.toLowerCase());
   }
 
+  compareFnName(a,b) {
+    return a.name.toLowerCase() > b.name.toLowerCase() ? 1 : (a.name.toLowerCase() === b.name.toLowerCase() ? 0 : -1)
+  }
+
+  compareFnUser(a,b) {
+    if(a.user){
+      return parseName(a.user) > parseName(b.user) ? 1 : (parseName(a.user) === parseName(b.user) ? 0 : -1);
+    }
+    return parseName(a) > parseName(b) ? 1 : (parseName(a) === parseName(b) ? 0 : -1);
+  }
+
```

Comment:
Maybe I am wrong but wouldn't it be better if the sorting would happen in the group-search-select.component/vo-search-select.component/... so it would be universal and we would have it sorted everywhere where we are using the selectors?
I think that we don't have use-case where sorted items in the selector wouldn't make sense. So it would probably make sense.